### PR TITLE
Change location.

### PR DESCRIPTION
**Pull request by bwinton**

### CODE REVIEW

**Review by aminalhazwani — COMMENTED** [review]
I would name the repo as the package name so that we have name parity, so photon-colors instead of photon-colors-tokens, what do you think?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,20 @@
 # Photon Colors Releases
 
+## [3.3.0] - 2018-05-??
```
Comment (aminalhazwani) [inline CHANGELOG.md]: Reminder to add the date before merge

**Comment by bwinton** [comment]
I wondered about that. If we do that, then there's no indication of which repos are for tokens and which aren't, so that's why I added in the `-tokens`... Should I remove it?

**Comment by aminalhazwani** [comment]
I think that having it in the description is enough 🙂 
screenshot

**Review by aminalhazwani — APPROVED** [review]
Let's do this!

**Pull request merged**